### ISSUE TITLE
RST formatting only available when mousing over the class/method

### ISSUE THREAD

**Issue by NightChips-dev** [microsoft/pylance-release#566]
## Environment data

-   Language Server version: 2020.11.0
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): 3.9

## Expected behaviour
Proper RST formatting.
![image](https://user-images.githubusercontent.com/68968630/98214373-16c7c380-1f4f-11eb-9d64-8604ea82e0c3.png)

## Actual behaviour
Doesn't format anything.
![image](https://user-images.githubusercontent.com/68968630/98213785-46c29700-1f4e-11eb-897b-fe81320c32d2.png)

## Code Snippet / Additional information

`__init__.py`
```python
class a:
    def __init__(self, name):
         self.name = name

```
`__init__.pyi`
```python
class a:
    """Generic docstring"""
    def __init__(self, name : str) -> None:
          """Creates 'a' object

          Args:
          ----------
          - name: The name of the object
          """
```
`main.py`
```python
from package import a
a();

```
## Logs
```
[FG] parsing: c:\Users\***\Documents\MEGAsync\Python\***\***\motor.py (18ms)
[FG] binding: c:\Users\***\Documents\MEGAsync\Python\***\***\motor.py (5ms)
Background analysis message: setFileOpened
Background analysis message: getDiagnosticsForRange
Background analysis message: getDiagnosticsForRange
Background analysis message: getSemanticTokens
[BG(1)] parsing: c:\Users\***\Documents\MEGAsync\Python\***\***\motor.py (79ms)
[BG(1)] binding: c:\Users\***\Documents\MEGAsync\Python\***\***\motor.py (2ms)
Background analysis message: getSemanticTokens
Background analysis message: analyze
[BG(1)] analyzing: c:\Users\***\Documents\MEGAsync\Python\***\***\motor.py ...
[BG(1)]   checking: c:\Users\***\Documents\MEGAsync\Python\***\***\motor.py (6ms)
[BG(1)] analyzing: c:\Users\***\Documents\MEGAsync\Python\***\***\motor.py (6ms)
Background analysis message: resumeAnalysis
```

**Comment by jakebailey** [comment]
We were missing the docstring conversion for the signature provider; I'll get that fixed.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.11.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020111-11-november-2020
